### PR TITLE
Changes for devnet launch

### PR DESCRIPTION
**Pull request by DaMandal0rian**
- hardcode IP's into default
- launch new devnet

### CODE REVIEW

File: resources/devnet/main.tf [hunk 1]
```diff
@@ -9,9 +9,9 @@ module "devnet" {
     nodes-per-region    = 0
     additional-node-ips = var.bootstrap_node_ips
     docker-org          = "subspace"
-    docker-tag          = "snapshot-2023-may-06"
+    docker-tag          = "gemini-3d-2023-may-23"
```
Comment (vedhavyas) [inline resources/devnet/main.tf]: nope. this should be snapshot not gemini-3d

**Review by vedhavyas — COMMENTED** [review]
Snapshots are used for devnet. For gemini-3d we use images with `gemini-3d` prefix

**Review by vedhavyas — APPROVED** [review]

**Comment by vedhavyas** [comment]
@DaMandal0rian I'm not seeing docker image change. Is this coming in different PR ?

**Comment by DaMandal0rian** [comment]
> @DaMandal0rian I'm not seeing docker image change. Is this coming in different PR ?

@vedhavyas This the old framework and won't be used anymore. We have migrated.

**Pull request merged**